Type conversion with equality operator (#31706)
hashicorp/terraform: website/docs/language/expressions/types.mdx

> Where possible, Terraform automatically converts values from one type to
> another in order to produce the expected type. If this isn't possible, Terraform
> will produce a type mismatch error and you must update the configuration with a

Reviewer: ```suggestion
Where possible, Terraform automatically converts values from one type to
another to produce the expected type. If this conversion is impossible, Terraform produces a type mismatch error and you must update the configuration with a
```

Suggest changing " If this isn't possible, Terraform will produce" to "If this conversion is impossible, Terraform produces" to avoid "This" without a subject, to avoid contractions, and to avoid future tense.
